deno/payloads/v10/soundboard.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/resources/soundboard
+ */
+
+import type { Snowflake } from '../../globals.ts';
+import type { APIUser } from './user.ts';
+
+/**
+ * https://discord.com/developers/docs/resources/soundboard#soundboard-sound-object
+ */
+export interface APISoundboardSound {
+	/**
+	 * The name of this sound
+	 */
+	name: string;
+	/**
+	 * The id of this sound
+	 */
+	sound_id: Snowflake;
+	/**
+	 * The volume of this sound, from 0 to 1
+	 */
+	volume: number;
+	/**
+	 * The id of this sound's custom emoji
+	 */
+	emoji_id: Snowflake | null;
+	/**
+	 * The unicode character of this sound's standard emoji
+	 */
+	emoji_name: string | null;
+	/**
+	 * The filename of this sound (for default sounds)
+	 */
+	override_path: string | null;
```

Comment:
The CDN route that lets you use this is not added in the PR

Reply:
That property and CDN route are no longer documented, removed before the PR was merged

Reply:
https://discord.com/developers/docs/resources/soundboard#sound-files

Reply:
I added that one [here](https://github.com/discordjs/discord-api-types/pull/1113/files#diff-d94c4361c9ef564dcc1f291e8fa6742b21bfe7f75fb1642f39c7928fe034de4aR1357) but I don't see anything about `override_path` anymore

Reply:
Woops, I am as we say, le blind